bitcoin-magazine/bitcoin-explained/bitcoin-core-22.0-explained.md

```diff
@@ -0,0 +1,867 @@
+
+Aaron: 00:19:19
+
+Yeah.
+I think another way to put it short, if you, if you agree with me, is that there's no actual benefit yet in using Taproot in this Bitcoin Core release, but it's possible to use it for sort of regular transactions, right?
+
+Sjors: 00:19:31
+
+Yeah, exactly.
+So you can if you really know what you're doing.
+And other than that, it's probably useful to have it in there anyway for people who want to test things on SIGNET or TESNET or you know that sort of stuff.
+Because we want other developers to keep working on Taproot.
+
+Aaron: 00:19:44
+
+Yeah, so this was basically Taproot addresses we were discussing.
+A little bit deeper under the hood, you have the taproot descriptors.
+Descriptors?
+What are taproot descriptors?
+
+Sjors: 00:19:55
+
+Yes, so a descriptor is a way to say I want to generate a series of addresses, basically.
+So instead of saying, okay, our wallet is one address, you say, okay, our wallet starts with the root private key, and then it does a bunch of BIP32 style derivations, and then it says, okay, give me the address number 15 or 16 or 17.
+And descriptors allow you to describe that.
+And now they also let you do that for Taproot.
+
+Aaron: 00:20:25
+
+I think descriptors also allow you to sort of categorize your funds into specific types of funds, right?
+So you
+
+Sjors: 00:20:34
+
+Hmm hmm
+
+Aaron: 00:20:34
+
+can say, these are all my Taproot funds, locked in Taproot stuff, and these are all my Multisig funds, like the Multisig stuff, sorry, Multisig addresses.
+And the descriptors sort of let you label these, right?
+
+Sjors: 00:20:48
+
+Yeah, because they describe in a sort of a human readable form, what your addresses are, how to get to your addresses.
+Yeah.
+You can say, I have one descriptor for my SegWit addresses.
+I have one descriptor for my Taproot addresses.
+
+Aaron: 00:21:00
+
+Yeah.
+So now there's a Taproot descriptor.
+So that's another sort of small step in the Taproot process for Bitcoin Core.
+
+Sjors: 00:21:07
+
+Well, in fact, that is what the wallet has support for.
+So you can add Taproot descriptors to the wallet now, but you have to do it yourself.
+
+Aaron: 00:21:15
+
+Right.
+So, hardware wallet support in the GUI, I2P support, Taproot.
+
+Sjors: 00:21:23
+
+The other thing is there's a little safety measure there.
+Well, two things.
+So there's a little safety measure there that you cannot add Taproot to a wallet on mainnet before it activates.
+Just to make sure you don't have people sent to your taproot address before it activates because then you lose all the coins.
+And the other thing is that there's now some...
+
+Aaron: 00:21:45
+
+Wait, wait, Wait, why would you lose the coins?
+It just means you can't spend the coins, right?
+
+Sjors: 00:21:48
+
+No, if you're receiving money on a Taproot address before Taproot activates, then that's anyone can spend.
+
+Aaron: 00:21:55
+
+Yeah, right.
+Right.
+
```

Comment:
Another example of a line that doesn't contribute to the transcript's content - you can just delete this and continue with the same speaker.